packages/http/src/resources/Hotspots.ts

```diff
@@ -60,4 +60,14 @@ export default class Hotspots {
     } = await this.client.get(url)
     return new Hotspot(this.client, hotspot)
   }
+
+  async elected(block?: number): Promise<Hotspot[]> {
```

Comment:
Does this us a cursor? Usually we follow the ResourceList pattern for array to support cursors in the API and @allenan asked me to do that even for non cursor APIs and then we can just send a MAX value. 

Should we update this one to use ResourceList as well @allenan

Reply:
ah okay yeah was considering that but wasn't sure if it made sense here or not

Reply:
yeah if we return a list of resources, we should use the `ResourceList` class so that the iteration methods exposed are consistent